[PREVIEW DRAFT] Update enhancements handbook

palnabarun opened the pull request.

Commit:
Revise docs release handbook

- Number the steps
- Format as shell
- Add advice on continuing after a merge conflict

Commit:
Drop exception for Chinese localization branch version

palnabarun committed:
Refactor Enhancements Role handbook to replace instances of SIG PM

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

palnabarun committed:
Refactor the timeline format in the Enhancements Role handbook

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

Commit:
patch releases: fix off by one in 1.16 branch info

1.16.15 has not been released yet, but it will be the last release for
the branch.

Signed-off-by: Tim Pepper <tpepper@vmware.com>

Commit:
patch releases: add 1.19 branch

Add targets for 1.19.1 in keeping with prior cadences.

Signed-off-by: Tim Pepper <tpepper@vmware.com>

Commit:
patch releases: remove 1.16 at EOL

As of the 1.16.15 release, the release-1.16 branch is no longer
receiving community support.

Signed-off-by: Tim Pepper <tpepper@vmware.com>

Commit:
patch releases: align 1.19.2 with monthly cadence

As in the past, the 1.XX.1 release follows in a week or two after the
1.XX.0 release, and then we shift onto a monthly patch release branches
combined release date target.

Signed-off-by: Tim Pepper <tpepper@vmware.com>

kikisdeliveryservice commented on release-team/role-handbooks/enhancements/README.md:
```diff
@@ -64,50 +61,73 @@ The shadows should be selected keeping in mind that one of them may eventually b
+
+#### Pre-Collection (Week 0)
+
+- Gather Shadows to have them read this handbook, watch the [k/enhancements][k/enhancements] repo to get notified on all issues, and give expectations on what the process looks like and their particular role.
```
since shadows are selected later maybe move this to week 1?

palnabarun replied:
Done! :+1:

k8s-ci-robot committed:
Merge pull request #1209 from tpepper/master

patch release schedule updates

kikisdeliveryservice commented on release-team/role-handbooks/enhancements/README.md:
```diff
@@ -64,50 +61,73 @@ The shadows should be selected keeping in mind that one of them may eventually b
+- Examine the `Removed From Milestone` tab and visit those issues to see if they have been added to the current milestone. These enhancements likely slipped because of docs or testing. Ping owners to make sure they are targeted for the current milestone. When complete, clear the issues from the tab.
+- Find Issues from previous milestone that have graduated to Stable. Remove `tracked/yes` or `tracked/no` labels. Check to see if the KEP status has been updated to `implemented`. If it has, close the issue. If it has not, ask the issue contact to both update the KEP status field and close the Enhancement issue once the update PR has merged.
+- Clear out previous milestone issues. Ping owners to get them to drop milestone or add to the current milestone so it can be tracked. This can be a responsibility for shadow roles.
+- Close previous milestone.
```
How does one do that?

palnabarun replied:
Clarified the task. It's ensuring that there are no open issues/PRs in the milestone.

kikisdeliveryservice commented on release-team/role-handbooks/enhancements/README.md:
```diff
@@ -64,50 +61,73 @@ The shadows should be selected keeping in mind that one of them may eventually b
+
+- Stay on top of comments in issues when owners respond. Apply correct labels, milestone information, update their status in the sheet if necessary.
+- Mark features as `At Risk` if there is no communication, active PRs on the issues, or it is missing other requirements coming into Enhancement Freeze.
+- Start syncing with Communications Team on giving an induction what's coming up for the release.
```
induction -> indication/signal

palnabarun replied:
It was actually meant to be an induction. The enhancements lead/team is supposed to sync up with the comms lead to give a brief about the important enhancements in the milestone.

kikisdeliveryservice commented on release-team/role-handbooks/enhancements/README.md:
```diff
@@ -64,50 +61,73 @@ The shadows should be selected keeping in mind that one of them may eventually b
+
+#### Post Enhancements Freeze (Week 5-10)
+
+- Stay on top of issues and continually monitor them twice a week and look at attached PRs. As Code Freeze gets closer, there are PRs that have not been merged, move the issue to At Risk. If there is no activity, ping issue owners on either the issue or the k/k PR.
```
"if there are PRs.."

palnabarun replied:
Done. :+1:

kikisdeliveryservice commented on release-team/role-handbooks/enhancements/README.md:
```diff
@@ -64,50 +61,73 @@ The shadows should be selected keeping in mind that one of them may eventually b
+#### Post Enhancements Freeze (Week 5-10)
+
+- Stay on top of issues and continually monitor them twice a week and look at attached PRs. As Code Freeze gets closer, there are PRs that have not been merged, move the issue to At Risk. If there is no activity, ping issue owners on either the issue or the k/k PR.
+- Monitor issues that are At Risk closer, almost daily. Code Freeze means no new code and keeping tabs on the status of the k/k PR is critical to planning. Make decisions if the enhancement should be deferred and work with SIG Leads to determine the best path forward.
```
closer-> closely

palnabarun replied:
Done. :+1:

kikisdeliveryservice reviewed (commented):
Looks great! Just a few comments on my end

savitharaghunathan committed:
Docs role handbook improvements

Commit:
patch releases: yaml update

This updates the yaml, but kubernetes/release repo's
cmd/schedule-builder is not used to create the markdown in the prior
commits as it doesn't quite handle the phasing of information
advertising we do at the start (and end) of a release branch's lifetime.

Signed-off-by: Tim Pepper <tpepper@vmware.com>

k8s-ci-robot committed:
Merge pull request #1210 from tpepper/schedule

patch release schedule updates v2

puerco committed:
Handbook updates for the new release notes process

Signed-off-by: Adolfo García Veytia (Puerco) <github@pog.eml.mx>

palnabarun committed:
Improve sections of the Enhancements handbook

- Move shadow gathering to Week 1
- Clarify closing milestone
- Change access required block

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

palnabarun committed:
Remove the lifecycle/frozen line since k/enhancements can't be frozen

palnabarun committed:
Add some tips on determining stage status

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

savitharaghunathan committed:
Adding prow commands link to RT onboarding doc

k8s-ci-robot committed:
Merge pull request #1214 from savitharaghunathan/rt_onboarding_update

Updating RT onboarding doc with a link to prow commands

cpanato committed:
patch-release: fix next 1.19 patch release

k8s-ci-robot committed:
Merge pull request #1219 from cpanato/fix-patch-1.19

patch-release: fix next 1.19 patch release

kikisdeliveryservice commented on release-team/role-handbooks/enhancements/README.md:
```diff
@@ -37,17 +34,17 @@ An Enhancements Lead holds the following responsibilities:
-Beginning of the cycle through enhancement freeze: 3-6 hours a week fluctuating based on how many SIG meetings need to be attended
-Enhancement Freeze through Code Freeze: 4-7 hours a week
-Code Freeze through Release Day: 1-4 hours a week
+- Beginning of the cycle through enhancement freeze: 3-6 hours a week fluctuating based on how many SIG meetings need to be attended
```
Feels like 3-6 is a little low during prime time?

palnabarun replied:
I agree with that assessment. It's usually more than those many hours.

I feel 6-10hours could be a good ballpark. What do you think?

kikisdeliveryservice reviewed (commented):
Looks great- 1 non-blocking q

justaugustus committed:
docs-handbook: Apply suggestions from code review

k8s-ci-robot committed:
Merge pull request #1014 from sftim/20200307_update_docs_release_handbook

Update SIG Docs release handbook

justaugustus committed:
release-managers: Lint doc

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
release-managers: Update SIG Release Leads section

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
release-managers: Promote Marko Mudrinić to full-fledged Release Manager

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
release-managers: Remove Marky Jackson from team

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
release-managers: Update Associates

- Adolfo García Veytia (new)
- Gianluca Arbezzano (new)
- Max Körbächer (new)
- Taylor Dolezal (returning)

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

k8s-ci-robot committed:
Merge pull request #1225 from justaugustus/relmgr

Update Release Managers membership

kikisdeliveryservice committed:
update README.md to clarify enhancement sheet access

Also remove refs to defunct SIG PM and use new
release-team@kubernetes.io group.

kikisdeliveryservice committed:
update sig-leads-group to use leads@kubernetes.io

k8s-ci-robot committed:
Merge pull request #1228 from kikisdeliveryservice/update-enhancements-sheet-instructions

enhancements: update README.md to clarify enhancement sheet access

palnabarun committed:
Update Kubernetes Release Team email

Replace occurences of kuberentes-release-team@googlegroups.com
with release-team@kubernetes.io

Ref: https://github.com/kubernetes/k8s.io/pull/1250

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

jeremyrickard committed:
Fix filename

Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

k8s-ci-robot committed:
Merge pull request #1230 from jeremyrickard/fix-release-team-filename

Fix filename

palnabarun committed:
Update all references to the old Kubernetes Release Team Google Group

Replace https://groups.google.com/forum/#!forum/kubernetes-sig-release
with https://groups.google.com/a/kubernetes.io/g/release-team and
https://groups.google.com/forum/#!forum/kubernetes-sig-release-shadows
with https://groups.google.com/a/kubernetes.io/g/release-team-shadows

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

k8s-ci-robot committed:
Merge pull request #1232 from palnabarun/update-release-team-email

Update references to the Kubernetes Release Team and Kubernetes Release Team Shadows Google Group

annajung committed:
Update release docs role handbook to reflect change to the default channel of communication

k8s-ci-robot committed:
Merge pull request #1233 from annajung/master

Update release docs role handbook to reflect correct channel

kikisdeliveryservice committed:
add 1.20 enhancements tracking sheet link

k8s-ci-robot committed:
Merge pull request #1234 from kikisdeliveryservice/add-120-enhancements-sheet

releases/release-1.20: add 1.20 enhancements tracking sheet link

kikisdeliveryservice committed:
fix enhancements sheet link

k8s-ci-robot committed:
Merge pull request #1235 from kikisdeliveryservice/fix-enhancements-sheet-link

fix 1.20 enhancements sheet link

jeremyrickard committed:
Add checkbox for slack group user mgmt

Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

savitharaghunathan committed:
Updating RT & RT shadows google group references

k8s-ci-robot committed:
Merge pull request #1236 from jeremyrickard/upate-rt-access-template

Add checkbox for slack group user mgmt

k8s-ci-robot committed:
Merge pull request #1237 from savitharaghunathan/update_dl

Updating RT & RT shadows google group references

cpanato committed:
patches: update patch release schedule for 1.19/1.18/1.17

spiffxp committed:
Clarify test freeze defintion

k8s-ci-robot committed:
Merge pull request #1243 from spiffxp/clarify-test-freeze

Clarify test freeze defintion

hasheddan committed:
Fix missing release managers link

Fixes a broken link in patch release team doc to point at relevant
release managers doc.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

hasheddan committed:
Cut 1.20.0-alpha.1 release in 1.20 cycle

This modifies the 1.20 release schedule to start with 1.20.0-alpha.1
release instead of 1.20.0-alpha.2, and adjusts the following releases
accordingly.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

k8s-ci-robot committed:
Merge pull request #1245 from hasheddan/1.20-alphas

Cut 1.20.0-alpha.1 release in 1.20 cycle

jeremyrickard committed:
Propose test freeze deadline

Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

k8s-ci-robot committed:
Merge pull request #1212 from savitharaghunathan/docs_handbook_update

Docs role handbook improvements

k8s-ci-robot committed:
Merge pull request #1244 from hasheddan/rel-mgr-link

Fix missing release managers link

annajung committed:
Cleanup docs handbook

annajung committed:
Add 1.20 Docs Shadows to the release team

k8s-ci-robot committed:
Merge pull request #1250 from annajung/docs-add-shadows

Add 1.20 Docs Shadows to the release team

k8s-ci-robot committed:
Merge pull request #1242 from cpanato/update-patch

patches: update patch release schedule for 1.19/1.18/1.17

bai committed:
Add 1.20 Release Team Bug Triage Shadows

RobertKielty committed:
Add 1.20 release CI Sginal shadows

Signed-off-by: RobertKielty <rob.kielty@gmail.com>

k8s-ci-robot committed:
Merge pull request #1252 from bai/add-bug-triage-shadows

Add 1.20 Release Team Bug Triage Shadows

k8s-ci-robot committed:
Merge pull request #1254 from RobertKielty/1.20cisigshadows

Add 1.20 release CI Signal shadows

hasheddan committed:
Cleanup cutting release section in branch manager handbook

Fixes up some links to be more readable and clarifies on language in
steps for cutting releases.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

k8s-ci-robot committed:
Merge pull request #1255 from hasheddan/alpha-link

Cleanup cutting release section in branch manager handbook

markjacksonfishing committed:
releng: Add Marky to Release Manager Associates

kikisdeliveryservice committed:
add 1.20 Enhancements shadows

jrsapi committed:
Add Comms Shadows contact info for v. 1.20

k8s-ci-robot committed:
Merge pull request #1260 from jrsapi/add-comms-shadows-1_20

Add Comms Shadows contact info for v. 1.20

k8s-ci-robot committed:
Merge pull request #1248 from annajung/master

Docs role handbook clean up

k8s-ci-robot committed:
Merge pull request #1259 from kikisdeliveryservice/add-enhancements-shadows

release-1.20/release_team.md: add 1.20 Enhancements shadows

k8s-ci-robot committed:
Merge pull request #1258 from markyjackson-taulia/master

releng: Add Marky to Release Manager Associates

JamesLaverack committed:
Add 1.20 Release Notes shadows

k8s-ci-robot committed:
Merge pull request #1261 from JamesLaverack/1.20-release-notes-shadows

Add 1.20 Release Notes shadows

k8s-ci-robot committed:
Merge pull request #1246 from jeremyrickard/add-test-freeze-date-1.20

Propose test freeze deadline for 1.20

hasheddan committed:
Add team meeting with shadows to RT Lead handbook

Adds a pre-release cycle activity to RT Lead handbook to schedule a
group meeting with shadows to establish responsibilities and timeline so
that they can assist in onboarding activities for other RT members.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

hasheddan committed:
Add image promotion step to release cut issue template

Adds image promotion step and krel command for opening PR against k8s.io
with images for release version. This step is required for all releases
since VDF.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

k8s-ci-robot committed:
Merge pull request #1263 from hasheddan/rt-lead-meet

Add team meeting with shadows to RT Lead handbook

k8s-ci-robot committed:
Merge pull request #1264 from hasheddan/iss-prom-img

Add image promotion step to release cut issue template

jeremyrickard committed:
Add Xander as 1.20 Comms Shadow

Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

k8s-ci-robot committed:
Merge pull request #1265 from jeremyrickard/onboard-xander-shadow

Add Xander as 1.20 Comms Shadow

kcmartin committed:
Fetch all upstream branches

Clarify that one needs to fetch all branches from upstream website repo, not just master branch

annajung committed:
Add access details and clean up wrong url in docs role handbook

jeremyrickard committed:
Cleanup OWNER_ALIASES

Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

kcmartin committed:
Adding new step to fetch dev-branch

Per PR discussion we don't really need to fetch all branches

k8s-ci-robot committed:
Merge pull request #1266 from kcmartin/patch-1

Clarify instructions to fetch all upstream branches

k8s-ci-robot committed:
Merge pull request #1268 from annajung/master

Add access details and clean up wrong url in docs role handbook

k8s-ci-robot committed:
Merge pull request #1267 from jeremyrickard/cleanup-owner-alias

Cleanup OWNER_ALIASES

JamesLaverack committed:
Release Notes draft for k/k v1.20.0-alpha.1

Signed-off-by: James Laverack <james.laverack@jetstack.io>

k8s-ci-robot committed:
Merge pull request #1269 from JamesLaverack/release-notes-draft-v1.20.0-alpha.1

Update release notes draft to version v1.20.0-alpha.1

jeremyrickard committed:
Update onboarding issue and RT Lead Handbook for Inclusive Speaker Orientation

Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

Commit:
Release Notes draft for k/k v1.20.0-alpha.1

Signed-off-by: Celeste Horgan <celeste@cncf.io>

k8s-ci-robot committed:
Merge pull request #1273 from celestehorgan/release-notes-draft-v1.20.0-alpha.1

Update release notes draft to version v1.20.0-alpha.1

kikisdeliveryservice committed:
add 1.20 exceptions.yaml

palnabarun committed:
Add exceptions for the 1.19 release cycle

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

k8s-ci-robot committed:
Merge pull request #1275 from palnabarun/retro-add-1-19-exceptions

Add exceptions for the 1.19 release cycle

k8s-ci-robot committed:
Merge pull request #1270 from jeremyrickard/add-cncf-unconscious-bias-requirement

Update onboarding issue and RT Lead Handbook for Inclusive Speaker Orientation

xmudrii committed:
OWNERS_ALIASES: Add xmudrii to release-engineering-approvers

k8s-ci-robot committed:
Merge pull request #1276 from xmudrii/update-owners

OWNERS_ALIASES: Add xmudrii to release-engineering-approvers

k8s-ci-robot committed:
Merge pull request #1274 from kikisdeliveryservice/add-120-exceptionsyaml

add 1.20 exceptions.yaml

xmudrii committed:
Update the branch manager role handbook with additional details

xmudrii committed:
Address review comments

xmudrii committed:
Add a TODO to remove cip-mm requirement

k8s-ci-robot committed:
Merge pull request #1277 from xmudrii/role-handbook-update

Update the branch manager role handbook with additional details

kikisdeliveryservice committed:
update exceptions.yaml to incl exceptions 10-12 through 10-13

k8s-ci-robot committed:
Merge pull request #1285 from kikisdeliveryservice/update-exceptions

update exceptions.yaml to incl exceptions 10-12 through 10-13

lasomethingsomething committed:
Removed reference to Velodrome

Velodrome no longer exists.

k8s-ci-robot committed:
Merge pull request #1286 from kubernetes/LappleApple-patch-1

Removed reference to Velodrome

lasomethingsomething committed:
Update to Release Eng Schedule of Recurring Work

Adding the relevant results from our Ways of Working discussions in Rel Eng.

annajung committed:
Add template for reaching out to PR owners for doc deadlines

annajung committed:
Separate release timeline into its own doc for readability

k8s-ci-robot committed:
Merge pull request #1288 from LappleApple/master

Update to Release Eng Schedule of Recurring Work

lasomethingsomething committed:
Updating Release Managers Roles

Adding the relevant results from our Ways of Working discussions in Rel Eng.

lasomethingsomething committed:
Small edits in Associates section

Needed to make the grammar consistent.

k8s-ci-robot committed:
Merge pull request #1292 from LappleApple/master

Updating the Release Engineering Roles Descriptions

lasomethingsomething committed:
Deleted Velodrome references from handbook

Velodrome is gone.

k8s-ci-robot committed:
Merge pull request #1293 from LappleApple/master

Deleted Velodrome references from handbook

cpanato committed:
patches: update calendar for the next patches releases

Commit:
patch release schedule: add expected EOL for 1.18, 1.17

As a part of WG LTS's wind down, we've discussed that given expected
release cadences that 1.18 and 1.17 are going to get a year of patch
support.  It's been noted though that the schedule remained ambiguous
for these two.  This commit adds an EOL target date in the month
of their release one year anniversary.

Signed-off-by: Tim Pepper <tpepper@vmware.com>

k8s-ci-robot committed:
Merge pull request #1294 from cpanato/update-patches

patches: update schedule for the next patches releases

annajung committed:
Add branch specification to the template

wilsonehusin committed:
Release notes for 1.20.0-alpha.2

k8s-ci-robot committed:
Merge pull request #1299 from wilsonehusin/release-notes-draft-v1.20.0-alpha.2

Update release notes draft to version v1.20.0-alpha.2

k8s-ci-robot committed:
Merge pull request #1297 from tpepper/schedule

patch release schedule: add expected EOL for 1.18, 1.17

xmudrii committed:
Move 1.20 branch/jobs creation to rc.0 and replace rc.1 with rc.0

k8s-ci-robot committed:
Merge pull request #1300 from xmudrii/schedule-1.20

Move 1.20 branch/jobs creation to rc.0 and replace rc.1 with rc.0

k8s-ci-robot committed:
Merge pull request #1289 from annajung/master

Doc role handbook clean up

justaugustus committed:
OWNERS: Update Build Admins

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
OWNERS: Update Release Team Leads

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
OWNERS: Update Release Engineering reviewers/approvers

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

k8s-ci-robot committed:
Merge pull request #1303 from justaugustus/owners

Update OWNERS

palnabarun committed:
Increase the time commitment estimation for pre-enhancements freeze period

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

wilsonehusin committed:
Release Notes draft for k/k v1.20.0-alpha.3

Signed-off-by: Wilson E. Husin <wilsonehusin@gmail.com>

lasomethingsomething committed:
Add GitHub handle to Associates section

This enhances standardisation.

k8s-ci-robot committed:
Merge pull request #1307 from LappleApple/master

Add GitHub handle to Associates section

thejoycekung committed:
Add note about holding an onboarding meeting for shadows

k8s-ci-robot committed:
Merge pull request #1308 from thejoycekung/onboard_meetings

Update CI Signal handbook to include holding an onboarding meeting for shadows

lasomethingsomething committed:
Update README.md

lasomethingsomething committed:
Update README.md

k8s-ci-robot committed:
Merge pull request #1310 from LappleApple/master

Adding Ways of Working Content

lasomethingsomething committed:
Included "Tracking Progress" section

Now posted to main page.

lasomethingsomething committed:
Include "Career Ladder" info for RMs and RMAs

Also details involvement in Buddy program and fixes a few typos.

k8s-ci-robot committed:
Merge pull request #1311 from LappleApple/master

Add "Tracking Work" section and Rel Eng Career Ladder info

jeremyrickard committed:
Remove Shubham from release-team.md

Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

k8s-ci-robot committed:
Merge pull request #1314 from jeremyrickard/update-release-team-file

Remove Shubham from release-team.md

k8s-ci-robot committed:
Merge pull request #1305 from wilsonehusin/release-notes-draft-v1.20.0-alpha.3

Update release notes draft to version v1.20.0-alpha.3

spiffxp committed:
Use do-not-merge/needs-foo labels for kubernetes/kubernetes

k8s-ci-robot committed:
Merge pull request #1317 from spiffxp/do-not-merge-needs-foo

Use do-not-merge/needs-foo labels for kubernetes/kubernetes

xmudrii committed:
Add additional information about sending announcement

xmudrii committed:
Add note about additional auth needed for krel gcbmgr history

xmudrii committed:
Add information about Skopeo

k8s-ci-robot committed:
Merge pull request #1213 from puerco/handbook-update

Release notes role handbook update detailing new process

Commit:
Update release notes handbook with onboarding info

k8s-ci-robot committed:
Merge pull request #1320 from tanjacky/update-release-notes-handbook

Update release notes handbook with onboarding info

k8s-ci-robot committed:
Merge pull request #1319 from xmudrii/update-branch-management-handbook

Update release branch management handbook

hasheddan committed:
Update CI Signal Issue format to be more conducive to triage

This updates the documentation on openining issues for the CI Signal
team based on feedback from SIGs. The primary changes include
centralizing all information for opening issues to a single section, as
well as providing a decision tree that informs a team member whether
they should open one or multiple issues, and how each issue should be
named.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

k8s-ci-robot committed:
Merge pull request #1163 from hasheddan/ci-issues

Update CI Signal Issue format to be more conducive to triage

JamesLaverack committed:
Release Notes draft for k/k v1.20.0-beta.1

Signed-off-by: James Laverack <james.laverack@jetstack.io>

k8s-ci-robot committed:
Merge pull request #1323 from JamesLaverack/release-notes-draft-v1.20.0-beta.1

Update release notes draft to version v1.20.0-beta.1

annajung committed:
Add a step about modifying prow config file to automatically apply milestones

palnabarun committed:
release-1.20: add meeting update template

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

palnabarun committed:
release-1.20: add 2020-11-04 meeting updates

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

palnabarun committed:
release-1.20: add 2020-11-06 meeting updates

Co-authored-by: Jeremy Rickard <rickardj@vmware.com>

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

k8s-ci-robot committed:
Merge pull request #1326 from palnabarun/async-proto

Add Release 1.20 Meeting Updates template and updates for Nov 4 and Nov 6

sayanchowdhury committed:
Fix the Release Update templates for Bug Triage

Signed-off-by: Sayan Chowdhury <sayan.chowdhury2012@gmail.com>

k8s-ci-robot committed:
Merge pull request #1327 from sayanchowdhury/sayan/fix-bug-triage-template

Fix the Release Update template for Bug Triage

k8s-ci-robot committed:
Merge pull request #1325 from annajung/master

Add a step about modifying prow config file to automatically apply milestones

justaugustus committed:
Rotate leadership team listed on Release Managers page

- Tim steps down as SIG Chair
- Sascha steps up as SIG Chair
- Dan steps up as SIG Technical Lead

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
release-managers.md: Wrap lines at 80-char

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
releng: Promote Adolfo to full-fledged Release Manager

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
releng: Add Arnaud as a Release Manager Associate

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

justaugustus committed:
releng: Tim, Hannes, and Doug stepping down as Release Managers

ref: responses to the Release Managers check-in survey

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

cpanato committed:
handbook: update prow job name for the image promotion

k8s-ci-robot committed:
Merge pull request #1332 from cpanato/update-doc

handbook: update prow job name for the image promotion

cpanato committed:
patches: update calendar for 1.17/1.18/1.19 patches releases

k8s-ci-robot committed:
Merge pull request #1333 from cpanato/update-patch-schedule

patches: update calendar for 1.17/1.18/1.19 patches releases

xmudrii committed:
Remove Branch Management update from the template

k8s-ci-robot committed:
Merge pull request #1335 from xmudrii/update-1.20-meeting-notes-template

Remove Branch Management update from the template

palnabarun committed:
release-1.20: add 2020-11-09 meeting updates

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

palnabarun committed:
release-1.20: add 2020-11-11 meeting updates

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

palnabarun committed:
release-1.20: update the meeting template

Fixes the line break issues in Markdown.

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

k8s-ci-robot committed:
Merge pull request #1331 from justaugustus/rotate

Update Release Managers (promotions, new members, retirement, and inactivity updates)

palnabarun committed:
release-1.20: add 2020-11-13 meeting updates

Signed-off-by: Nabarun Pal <pal.nabarun95@gmail.com>

k8s-ci-robot committed:
Merge pull request #1336 from palnabarun/update-async-notes

Release 1.20: Update async meeting notes
